SEC/Debt/Bonds.rdf

```diff
@@ -1036,6 +1039,7 @@
 		<rdfs:label>treasury bill</rdfs:label>
 		<skos:definition>short-term zero coupon treasury obligation with a maturity ranging from one to twelve months</skos:definition>
 		<fibo-fnd-utl-av:abbreviation>T-bill</fibo-fnd-utl-av:abbreviation>
+		<fibo-fnd-utl-av:explanatoryNote>The pricing of T-Bills is unique among government debt issues. Rather than providing interest payments as Treasury Bonds or Notes do, T-Bills are sold at a discount, and the entire return is realized upon maturity. The interest rate earned on T-Bills is equal to the difference between the purchase price and maturity value, divided by the maturity value.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
a) should say "unique among US government debt issues"
b) should say that the interest rate is annualized (I assume)
c)  if it's tradable what's the price then?

Reply:
more on c: T-Bills are tradable.  "Some of the other ways to buy Treasuries include ETFs, money market accounts, and the secondary market." -- from Investopedia